lib/public/views/Environments/format/displayEnvironmentStatusHistory.js

```diff
@@ -0,0 +1,68 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+import { h } from '/js/src/index.js';
+
+/**
+ * Display the given environment's status history string
+ *
+ * @param {Environment} environment the environment for which status should be displayed
+ * @return {vnode} the resulting component
+ */
+export const displayEnvironmentStatusHistory = (environment) => {
+    const { historyItems } = environment;
+
+    const statusAcronyms = {
+        STANDBY: 'S',
+        DEPLOYED: 'DL',
+        CONFIGURED: 'C',
+        RUNNING: 'R',
+        ERROR: 'E',
+        MIX: 'M',
+        DESTROYED: 'DS',
+    };
+
+    /**
+     * Get correct color class from environmentname.
+     * @param {*} status One of the Environment statuses
+     * @returns {string} The classname used to color the acronoym.
+     */
+    const getColorClassEnvironment = (status) => {
```

Comment:
You should not add again the status but look to complete `ColoredEnvironmentStatusComponent.js`

Reply:
- ColoredEnvironmentStatusComponent has been completed to allow optional parameter called content. This will default to the status name, otherwise it takes given content.